tools/languagelocalizer/languageLocalizer.js

```diff
@@ -161,6 +161,15 @@ const LanguageLocalizer = {
       await this.localizeLanguage(language);
     }
 
+    // `localizeLanguage` creates a `translations` object with all the language
+    // translations in language X using WikiData as a data source (via SPARQL
+    // queries).  Now we use CLDR which will give us X translated in all the
+    // languages in an object called `languages`. We need to normalize these 2
+    // objects and then drop the `translations`. WikiData is used fallback.
```

Comment:
```suggestion
    // objects and then drop the `translations`. Wikidata is used as fallback.
```

Note across the patch: it's `Wikidata`, not `WikiData` or `wikidata`